express-post-json/index.js

```diff
@@ -0,0 +1,23 @@
+const express = require('express');
+const app = express();
+let nextId = 1;
+const grades = {};
+
+app.use(express.json());
+
+app.listen(3000, () => {
+});
+
+app.get('/api/grades', (req, res) => {
+  const gradesArray = [];
+  for (const id in grades) {
+    gradesArray.push(grades[id]);
+  }
+  res.json(gradesArray);
+});
+
+app.post('/api/grades', (req, res) => {
+  req.body.id = nextId;
+  grades[nextId] = req.body;
+  nextId++;
+});
```

Comment:
Be sure to also respond to the client by sending back the grade object (req.body in this case) now that it has an id

```js
app.post('/api/grades', (req, res) => {
  req.body.id = nextId;
  grades[nextId] = req.body;
  nextId++;
  res.status(201).json(req.body);
});
```